easybuild/easyconfigs/h/HOME/HOME-0.9_setup.patch

```diff
@@ -0,0 +1,14 @@
+diff -ru HOME-0.9/setup.py HOME-0.9.modified/setup.py
+--- HOME-0.9/setup.py	2018-04-16 13:10:44.000000000 +0000
++++ HOME-0.9.modified/setup.py	2018-04-24 18:28:10.856736000 +0000
+@@ -11,8 +11,8 @@
+     author = 'akanksha srivastava',
+     install_requires = [
+         'numpy',
+-        'pandas==0.17.1',
+-        'scipy==0.16.0',
++        'pandas',
++        'scipy',
+         'scikit-learn==0.16.1',
+         'statsmodels==0.6.1',
```

Comment:
@timeu Why not do the same for `scikit-learn` and `statsmodels`, there's little point in fixing those version that hard I think?